.github/workflows/ci.yml

```diff
@@ -5,6 +5,33 @@ on:
   pull_request:
 
 jobs:
+  check-style:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Check line length
+        run: |
+          $found = $false
+          Foreach ($pattern in "*.cs","*.tt") {
```

Comment:
QUESTION: Should we remove the `*.tt` pattern?

Reply:
@m4rs-mt only asked for `*.cs` originally, but I figured we should check the template files as well. Maybe we change the default line length for those? Or we add a way to make it configurable per file (like having a line starting with  `// max_line_length: xxx`)?